nsxt/resource_nsxt_policy_group.go

```diff
@@ -596,15 +686,21 @@ func resourceNsxtPolicyGroupCreate(d *schema.ResourceData, m interface{}) error
 		return err
 	}
 
+	extendedCriteriaSets := d.Get("extended_criteria").([]interface{})
```

Comment:
do you think we should add a condition for local manager only? Or maybe there should be a validation func that errors out if these are defined for GM.

Reply:
Sure. I think we can do both. Let me add a validation func here and add some explanation in doc.